tests/app/notifications/test_process_notifications.py

```diff
@@ -252,23 +252,24 @@ def test_persist_notification_increments_cache_if_key_exists(sample_template, sa
     mock_incr.assert_called_once_with(str(sample_template.service_id) + "-2016-01-01-count", )
 
 
-@pytest.mark.parametrize((
-    'research_mode, requested_queue, notification_type, key_type, expected_queue, expected_tasks'
-), [
-    (True, None, 'sms', 'normal', 'research-mode-tasks', [deliver_sms]),
-    (True, None, 'email', 'normal', 'research-mode-tasks', [deliver_email]),
-    (True, None, 'email', 'team', 'research-mode-tasks', [deliver_email]),
-    (True, None, 'letter', 'normal', 'research-mode-tasks', [letters_pdf_tasks.create_letters_pdf]),
-    (False, None, 'sms', 'normal', 'send-sms-tasks', [deliver_sms]),
-    (False, None, 'email', 'normal', 'send-email-tasks', [deliver_email]),
-    (False, None, 'sms', 'team', 'send-sms-tasks', [deliver_sms]),
-    (False, None, 'letter', 'normal', 'create-letters-pdf-tasks', [letters_pdf_tasks.create_letters_pdf]),
-    (False, None, 'sms', 'test', 'research-mode-tasks', [deliver_sms]),
-    (True, 'notify-internal-tasks', 'email', 'normal', 'research-mode-tasks', [deliver_email]),
-    (False, 'notify-internal-tasks', 'sms', 'normal', 'notify-internal-tasks', [deliver_sms]),
-    (False, 'notify-internal-tasks', 'email', 'normal', 'notify-internal-tasks', [deliver_email]),
-    (False, 'notify-internal-tasks', 'sms', 'test', 'research-mode-tasks', [deliver_sms]),
-])
+@pytest.mark.parametrize(
+    "research_mode, requested_queue, notification_type, key_type, expected_queue, expected_tasks",
+    [
+        (True, None, 'sms', 'normal', 'research-mode-tasks', [deliver_sms]),
+        (True, None, 'email', 'normal', 'research-mode-tasks', [deliver_email]),
+        (True, None, 'email', 'team', 'research-mode-tasks', [deliver_email]),
+        (True, None, 'letter', 'normal', 'research-mode-tasks', [letters_pdf_tasks.create_letters_pdf]),
+        (False, None, 'sms', 'normal', 'send-sms-tasks', [deliver_sms]),
+        (False, None, 'email', 'normal', 'send-email-tasks', [deliver_email]),
+        (False, None, 'sms', 'team', 'send-sms-tasks', [deliver_sms]),
+        (False, None, 'letter', 'normal', 'create-letters-pdf-tasks', [letters_pdf_tasks.create_letters_pdf]),
+        (False, None, 'sms', 'test', 'research-mode-tasks', [deliver_sms]),
+        (True, 'notify-internal-tasks', 'email', 'normal', 'research-mode-tasks', [deliver_email]),
+        (False, 'notify-internal-tasks', 'sms', 'normal', 'notify-internal-tasks', [deliver_sms]),
+        (False, 'notify-internal-tasks', 'email', 'normal', 'notify-internal-tasks', [deliver_email]),
+        (False, 'notify-internal-tasks', 'sms', 'test', 'research-mode-tasks', [deliver_sms]),
+    ]
+)
```

Comment:
see my comment above about IDs for parameterize :)

Reply:
I agree! I added ids to the test I wrote, but this should be parts of other tickets to add ids to relevant tests.